ccm_web/server/src/main.ts

```diff
@@ -47,6 +48,11 @@ async function bootstrap (): Promise<void> {
   const sessionStore = new SequelizeStore({ db: sequelize, tableName: 'session' })
   sessionStore.sync({ logging: (sql) => logger.info(sql) })
 
+  // this is to increase the size limit for request payload, which defaults to 100kb
+  // then the array element limit will be enforced by the class-validator in CreateSectionsDto
+  app.use(json({ limit: '5mb' }));
+  app.use(urlencoded({ extended: true, limit: '5mb' }));
```

Comment:
So the benefit of increasing limit this is that the Express server can process the large payload, which it can then validate? I guess that's something. It'd be nice to know if hitting `100K` is actually that common with CSV files (in any feature). 400 enrollment records is around `7K`, so linearly, that's like 5200. If we think people will accidentally upload files with 10,000 records, then we are giving them a marginally better error message. It just feels like a small edge case. What happens now in the frontend (if you circumvent the CSV record limits in the `client`) is that user will see a "request entity too large" message, which is worse than "60 (or 400) is the limit". I guess that's a long-winded way of saying, I don't think it's essential we add this, but it also shouldn't hurt anything.

Reply:
Comments feel a bit unnecessary. You could just say briefly `// Controls size limit of data in payload and URL`

Reply:
updated the comment line